src/pytest_workflow/workflow_file_tests.py

```diff
@@ -0,0 +1,43 @@
+"""All tests for workflow files"""
+
+from pathlib import Path
+from typing import List
+
+import pytest
+
+
+class WorkflowFilesTestCollector(pytest.Collector):
+    def __init__(self, name, parent, files: List[dict], cwd):
+        self.files = files
+        self.name = name
```

Comment:
Same comments about already being handled in `super` call, and its simplifications